extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -536,3 +545,189 @@ export function jsonParseError(error: string, line: number, column: number) { re
 export const moreInformation = localize('moreInformation', "More Information");
 export const addPackageReferenceMessage = localize('addPackageReferenceMessage', 'To use SQL bindings, ensure your Azure Functions project has a reference to {0}', sqlExtensionPackageName);
 export const addSqlBindingPackageError = localize('addSqlBindingPackageError', 'Error adding Sql Binding extension package to project');
+
+// publish tab deploy options
+export const IgnoreTableOptions: string = localize('DeployOptions.IgnoreTableOptions', "Ignore Table Options");
+export const IgnoreSemicolonBetweenStatements: string = localize('DeployOptions.IgnoreSemicolonBetweenStatements', "Ignore Semicolon Between Statements");
+export const IgnoreRouteLifetime: string = localize('DeployOptions.IgnoreRouteLifetime', "Ignore Route Lifetime");
+export const IgnoreRoleMembership: string = localize('DeployOptions.IgnoreRoleMembership', "Ignore Role Membership");
+export const IgnoreQuotedIdentifiers: string = localize('DeployOptions.IgnoreQuotedIdentifiers', "Ignore Quoted Identifiers");
+export const IgnorePermissions: string = localize('DeployOptions.IgnorePermissions', "Ignore Permissions");
+export const IgnorePartitionSchemes: string = localize('DeployOptions.IgnorePartitionSchemes', "Ignore Partition Schemes");
+export const IgnoreObjectPlacementOnPartitionScheme: string = localize('DeployOptions.IgnoreObjectPlacementOnPartitionScheme', "Ignore Object Placement On Partition Scheme");
+export const IgnoreNotForReplication: string = localize('DeployOptions.IgnoreNotForReplication', "Ignore Not For Replication");
+export const IgnoreLoginSids: string = localize('DeployOptions.IgnoreLoginSids', "Ignore Login Sids");
+export const IgnoreLockHintsOnIndexes: string = localize('DeployOptions.IgnoreLockHintsOnIndexes', "Ignore Lock Hints On Indexes");
+export const IgnoreKeywordCasing: string = localize('DeployOptions.IgnoreKeywordCasing', "Ignore Keyword Casing");
+export const IgnoreIndexPadding: string = localize('DeployOptions.IgnoreIndexPadding', "Ignore Index Padding");
```

Comment:
seems good to maintain at one place! I'll implement like that!

Reply:
Description for the options is now coming from the API along with the dac options! But the option display names display on UI are still in ADS code base and I can also get them from same API, But just felt what if the UI name required an update, it would be an easy change to update the constants here( and if diff extensions need diff name for the options). Let me know your thoughts @kisantia

Reply:
Created an item for Display option Names to get from the DacFx API. will have a separate PR for that!